lib/notion/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 module Notion
-  VERSION = '1.1.0'
-  NOTION_REQUEST_VERSION = '2022-02-22'
+  VERSION = '1.1.1'
+  NOTION_REQUEST_VERSION = '2022-06-28'
```

Comment:
From my local testing, it seems like the version upgrade is not required for this to work. I would rather batch this upgrade in a PR that addresses upgrade-specific changes. Would you mind reverting this change?

Reply:
Reverted the change.